src/components/posts/CollegeAdmissions/essay/EssayContainer.js

```diff
@@ -0,0 +1,128 @@
+import React, {useEffect, useReducer, useState} from "react";
+import WrittenEssay from "./WrittenEssay";
+import ScoreBar from "./ScoreBar";
+import WordSuggestionsBar from "./WordSuggestionsBar";
+import "./essay.css";
+
+//words reducer set-up
+
+//our wordSections
+//changeable sections will also have a displayed property
+const wordSections = [
+    {
+        plain: "This is our example"
+    },
+    {
+        changeable: ["essay", "activity", "game"],
+        scores: [10,20,30]
+    },
+    {
+        plain: "made by our devs! What's your favorite"
+    },
+    {
+        changeable: ["meal", "food", "dish"],
+        scores: [10,20,30]
+    },
+    {
+        plain: "to cook?"
+    },
+    {
+        plain: "I know that I love"
+    },
+    {
+        changeable: ["fried rice", "spam masubi", "sushi"],
+        scores: [10,20,30]
+    },
+    {
+        plain: "at any time."
+    },
+];
+
+//set the displayed property to the middle index of the changeableWords
+const defaultWords = wordSections.map(section => section.plain ? {...section} 
```

Comment:
Maybe doing a forEach on the `wordSections` to set `displayed: 1` would have smaller space complexity? Then we could save it as a function/arrow function and then use it later when we want to reset the words. Also is there a reason why we don't set `wordSections` to have the default `displayed` value from the start?